packages/alfa-css/src/alphabet.ts

```diff
@@ -657,6 +665,35 @@ function consumeToken(stream: Stream<number>): Token | null {
       if (startsValidEscape(char, stream)) {
         return consumeIdentLike(char, stream);
       }
+      break;
+
+    case Char.NumberSign:
+      stream.advance(1);
+      const next = stream.peek(0);
+      const next2 = stream.peek(1);
```

Comment:
Whenever more than one lookahead is made, I tend to name the variables `fst`, `snd`, `thd`, `fth`. etc.:

```ts
const fst = stream.peek(0);
const snd = stream.peek(1);
```

Reply:
Good idea.